External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Flags.cpp

```diff
@@ -39,6 +39,11 @@ constexpr std::array<uint32_t, 17> FlagOffsets = {
 };
 
 void OpDispatchBuilder::SetPackedRFLAG(bool Lower8, OrderedNode *Src) {
+  // Canonicalize flags early
+  // Could be invalidate if we had masked invalidation
+  // Need to be careful since LAHF doesn't store OF
```

Comment:
What do we need to be careful about?

Reply:
Updated with a better comment stating that this is only a partial flag set. So we need to calculate flags early.
Also moved in to the Lower8 section since otherwise it is fine to invalidate the deferred state.